src/tokenlists/balancer/tokens/hyperevm.ts

```diff
@@ -5,4 +5,12 @@ export default [
   '0x02c6a2fA58cC01A18B8D9E00eA48d65E4dF26c70', // feUSD
   '0xBE65F0F410A72BeC163dC65d46c83699e957D588', // USDp
   '0x9B3a8f7CEC208e247d97dEE13313690977e24459', // sUSDp
+  '0x1ecd15865d7f8019d546f76d095d9c93cc34edfa', // LIQD
+  '0x11735dbd0b97cfa7accf47d005673ba185f7fd49', // CATBAL
```

Comment:
CATBAL but not PURR ? :thinking: 
https://www.coingecko.com/en/categories/hyperevm-ecosystem